monitoring/grafana/grr_grafanalib_dashboards/dashboards_to_generate/fleetspeak.dashboard.py

```diff
@@ -0,0 +1,239 @@
+from grafanalib.core import Alert, AlertCondition, Dashboard, Graph, Heatmap, LowerThan, OP_AND, Row, RTYPE_SUM, Target, TimeRange, YAxes, YAxis
+from grr_grafanalib_dashboards.util import add_data_source
+from grr_grafanalib_dashboards.config import ACTIVE_PROCESSES_ALERTING_CONDITION
+
+dashboard = Dashboard(
+  title="Fleetspeak Servers Dashboard",
+  rows=[
+    Row(panels=[
+      Graph(
+        title="Number of Active Processes",
+        targets=[
+          Target(
+            expr='sum(up{job="fleetspeak"})',
+            legendFormat="Active Processes",
+          ),
+        ],
+        alert=Alert(
+          name="Number of Active Processes alert",
+          message="The number of active Fleetspeak Server processes is below {}".format(ACTIVE_PROCESSES_ALERTING_CONDITION),
+          alertConditions=[
+            AlertCondition(
+              Target(
+                expr='sum(up{job="fleetspeak"})',
+                legendFormat="Active Processes",
+              ),
+              timeRange=TimeRange("10s", "now"),
+              evaluator=LowerThan(ACTIVE_PROCESSES_ALERTING_CONDITION),
+              operator=OP_AND,
+              reducerType=RTYPE_SUM
+            )
+          ],
+        )
+      ),
+      Graph(
+        title="Sum of Process Memory Bytes (across all instances)",
+        targets=[
+          Target(
+            expr='sum(process_resident_memory_bytes{job="fleetspeak"})',
+            legendFormat="Resident Memory",
+          ),
+        ]
+      ),
+      Graph(
+        title="CPU Usage",
+        targets=[
+          Target(
+            expr='avg(rate(process_cpu_seconds_total{job="fleetspeak"}[30s])) * 100',
+            legendFormat="Average Process CPU Usage in %",
+          ),
+        ],
+        yAxes=YAxes(
+          left=YAxis(max=105)
+        ),
+      ),
+    ]
+    ),
+    Row(panels=[
+      Graph(
+        title="Datastore Latency per Operation",
```

Comment:
It's not clear whether latency is expressed in seconds or in something else. Let's indicate the unit of the Y axis somehow (preferably - not by mentioning it in title/legend, but by explicitly configuring the Y axis presentation ).